site/content/en/docs/tasks/run/statefulset.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Run StatefulSet"
+linkTitle: "StatefulSet"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a StatefulSet as a Kueue-managed workload.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running StatefulSets.
+
+We demonstrate how to support scheduling StatefulSets in Kueue based on the Plain Pod integration with the help of PodGroup,
+where StatefulSet is represented as a single independent workload.
+
+This guide is for [serving users](/docs/tasks#serving-user) that have a basic understanding of Kueue.
+For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+
+2. Follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
+   to learn how to enable the `v1/pod` integration and how to configure it using the `podOptions` field.
+   Also, ensure that you have the `v1/statefulset` integration enabled.
```

Comment:
I would move it to a separate step and add an example, similar to what we have on the Pods page.